addons/gamification/tests/test_challenge.py

```diff
@@ -59,6 +61,67 @@ def test_10_reach_challenge(self):
         badge_ids = self.env['gamification.badge.user'].search([('badge_id', '=', badge_id), ('user_id', '=', demo.id)])
         self.assertEqual(len(badge_ids), 1, "Demo user has not received the badge")
 
+    @mute_logger('odoo.models.unlink')
+    def test_20_update_all_goals_filter(self):
+        challenge = self.env.ref('gamification.challenge_base_discover')
+
+        # Enroll two internal and two portal users in the challenge
+        self.env.ref('base.demo_user0').copy()
```

Comment:
In't it possible to create a new test user to be independent from existing data/demo ?

Reply:
Sure